docs/src/main/sphinx/connector/hive.rst

```diff
@@ -623,6 +623,9 @@ Property Name                                           Description
 
 ``hive.metastore.glue.write-statistics-threads``        Number of threads for parallel statistic writes to Glue,
                                                         defaults to ``5``.
+
+``hive.metastore.glue.assume-canonical-partition-keys`` Enable optimized metastore partition fetching for non-string
+                                                        partition keys. Default is ``false``.
```

Comment:
same here

Reply:
> The documentation should say when this is ok to set and when it's not.

I am missing knowledge in this area. For which types would it be a bad idea to set the parameter to `true` ?
Is there any automated test showcasing the limitations?